android/quest/src/main/java/org/smartregister/fhircore/quest/ui/register/dashboard/DashboardScreen.kt

```diff
@@ -184,12 +184,6 @@ fun DashboardScreen(
                 color = GreyTextColor,
                 style = MaterialTheme.typography.h6
               )
-              Spacer(modifier = Modifier.height(4.dp))
-              Text(
-                text = "Mallapadi 1",
-                color = Color.Black,
-                style = MaterialTheme.typography.h5
-              )
```

Comment:
why is this in this release?

Reply:
Currently this needs to be manually updated in all sites, which was missed earlier. Removed it now.